.github/.syncignore

```diff
@@ -1,2 +1,2 @@
 CODEOWNERS
-workflows/update-dependencies-from-metadata.yml
+workflows/update-dependencies.yml
```

Comment:
If we delete the `update-dependencies.yml` workflow file, should we also remove this entry from the `syncignore`?

Reply:
We should leave it in `.github/.syncignore` so that the `github-config` sync doesn't re-add the file.